homeassistant/data_entry_flow.py

```diff
@@ -187,7 +187,7 @@ class FlowHandler:
     # Set by flow manager
     flow_id: str = None  # type: ignore
     hass: Optional[HomeAssistant] = None
-    handler: str
+    handler: str = None  # type: ignore
```

Comment:
`handler: Optional[str] = None` or if you don't like the name `Union[str, None]`. I agree that is not perfect and maybe they fix it in a future PEP, but now is the correct type Optional. To ignore it is somethings like we can disable it in general

Reply:
or `handler: str = ""` like we use with c

Reply:
Each other approach brings up a lot of painful things. There will be type errors, pylint errors or straight up tests failing :(